app/routes/postsRouter.ts

```diff
@@ -1,72 +1,82 @@
 'use strict';
-import {Router, Request, Response, NextFunction, response} from 'express';
-import {Posts} from "../orm/entities/Posts";
+import {Router, Request, Response, NextFunction} from 'express';
+import {Post} from "../orm/entities/Post";
 import {getRepository} from "typeorm";
+import {validate, IsInt, ValidationError} from "class-validator";
```

Comment:
Hmm double check imports of this file also. Looks like `IsInt` for example is not used